### PR TITLE
Add edition for roadmap name and description

### PR DESCRIPTION
**Pull request by IidaKainu**
Takes advantage of overviewContent. Not necessarily in align with the look of the settings-page, but gets the job done.

### CODE REVIEW

**Review by PatricKangasmaki — APPROVED** [review]
😄

File: frontend/src/pages/ConfigurationPage.tsx [hunk 1]
```diff
@@ -22,13 +24,27 @@ const RoadmapConfigurationPageComponent = ({
...
   const roadmapId = useSelector(chosenRoadmapIdSelector);
   const dispatch = useDispatch<StoreDispatchType>();
   const userType = getType(userInfo, roadmapId);
-
+  const [patchRoadmap] = apiV2.usePatchRoadmapMutation();
   const { data: integrations } = apiV2.useGetIntegrationsQuery(
     roadmapId ?? skipToken,
   );
+  const { data: roadmaps } = apiV2.useGetRoadmapsQuery();
+  const roadmap = roadmaps?.find(({ id }) => id === roadmapId);
```
Comment (juhoeerola) [inline frontend/src/pages/ConfigurationPage.tsx:36]:
```suggestion
  const { data: roadmap } = apiV2.useGetRoadmapsQuery(undefined, {
    skip: !roadmapId,
    ...selectById(roadmapId),
  });
```

Could select by id directly from the query, and skip when no id is present.

**Review by juhoeerola — CHANGES_REQUESTED** [review]
I think the settings page definitely needs some nicer look to it, and I'm not sure if the `OverviewContent` will fit to that. But I don't think it's worth speculating too much now.

The functionality seems solid 👍

**Comment by IidaKainu** [comment]
> Could select by id directly from the query, and skip when no id is present.


Neat, added

**Review by juhoeerola — APPROVED** [review]

**Pull request merged**